src/main/scala/firrtl/ExecutionOptionsManager.scala

```diff
@@ -344,13 +346,16 @@ trait HasFirrtlOptions {
 
   parser.opt[String]("compiler")
     .abbr("X")
-    .valueName ("<high|middle|low|verilog|sverilog|none>")
+    .valueName ("<high|middle|low|verilog|mverilog|sverilog|none>")
```

Comment:
I'm okay with this being a "compiler" alternative but I think we should consider making it an orthogonal argument. This is more akin to `-O0` while the default is `-O2`

Reply:
I'd alsl be in favor of a `-O` style optimization level flags and explicit `-fno-constant-propagation`, etc. in the future. 

I'd propose to kick that can down the road to the dependency API discussion. Or: if we have optimization levels, I'd prefer for those to not alias to concrete `Compiler`s. I'm open to other ideas, though.

Reply:
Yeah I think that makes sense